docs/ai-testbed/groq/examples.md

```diff
@@ -0,0 +1,97 @@
+# Examples
+
+## Running Llama-2 7B on the Groqrack
+
+Llama-2 7B requires all 9 nodes in a groqrack.
+
+### Checking status
+
+First, verify that there are no other PBS jobs using the groqrack. This can be done from either a groqrack node or a login node.
+```console
+qstat -wa
+```
+If `qstat -wa` shows any jobs active, do not attempt to start Llama2.
+
+Also check to see if anyone is connected to any of the nodes: This can be done from a either grokrack node or a login node. Logins on nodes just indicate a potential conflict; not an actual conflict. 
```

Comment:
```suggestion
Also, be sure to check if anyone is connected to any of the nodes. This can be done from a either Grokrack node or a login node. Logins on nodes just indicate a potential conflict; not an actual conflict. 
```

Reply:
the last sentence is a bit confusing to me--- can you add more details?